man/sd_bus_set_description.xml

```diff
@@ -91,11 +103,17 @@
     url="view-source:https://dbus.freedesktop.org/doc/dbus-specification.html#auth-mechanisms">Authentication
     Mechanisms</ulink> section of the D-Bus specification for details.</para>
 
+    <para><function>sd_bus_is_anonymous()</function> returns true if the bus connection allows anonymous
+    authenatication (in the sense described in previous paragraph).</para>
+
     <para><function>sd_bus_set_trusted()</function> sets the "trusted" state on the
     <parameter>bus</parameter> object. If true, all connections on the bus are
-    trusted and access to all privileged and unprivileged methods is granted.  This
+    trusted and access to all privileged and unprivileged methods is granted. This
     function must be called before the bus has been started.</para>
```

Comment:
Not related to your changes but I would change this to "before the bus is started".